picard/ui/profileeditor.py

```diff
@@ -139,7 +135,7 @@ def load(self):
         """Load initial configuration.
         """
         config = get_config()
-        self.profile_settings = config.profiles[self.SETTINGS_KEY]
+        self.profile_settings = deepcopy(config.profiles[self.SETTINGS_KEY])
```

Comment:
Why this change?

Reply:
Because changes within the dialog were cascading into the `config.profiles` even without the user clicking "Make It So!"

Reply:
I'll add a comment in the code explaining that.